source/Engine/Bytecode/StandardLibrary.cpp

```diff
@@ -10017,6 +9933,38 @@ VMValue Music_Alter(int argCount, VMValue* args, Uint32 threadID) {
 	AudioManager::AlterMusic(panning, speed, volume);
 	return NULL_VAL;
 }
+/***
+ * Music.GetLoopPoint
+ * \desc Gets the loop point of a music index, if it has one.
+ * \param music (Integer): The music index to get the loop point.
+ * \return Returns the loop point in samples, as an Integer value.
+ * \ns Music
+ */
+VMValue Music_GetLoopPoint(int argCount, VMValue* args, Uint32 threadID) {
+	CHECK_ARGCOUNT(1);
+	ISound* audio = GET_ARG(0, GetMusic);
+	if (!audio) {
+		return NULL_VAL;
+	}
+	return INTEGER_VAL(audio->LoopPoint);
+}
+/***
+ * Music.SetLoopPoint
+ * \desc Sets the loop point of a music index.
+ * \param music (Integer): The music index to set the loop point.
+ * \param loopPoint (Integer): The loop point, in samples.
+ * \ns Music
+ */
+VMValue Music_SetLoopPoint(int argCount, VMValue* args, Uint32 threadID) {
+	CHECK_ARGCOUNT(1);
+	ISound* audio = GET_ARG(0, GetMusic);
+	int loopPoint = GET_ARG(1, GetInteger);
+	if (!audio) {
+		return NULL_VAL;
+	}
+	audio->LoopPoint = loopPoint;
```

Comment:
Alongside the above this should accept `null` to mean `-1`, but not accept integers lower than zero